### PR TITLE
Tapestry recipe

### PR DESCRIPTION
**Pull request by johnomics**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by johnomics** [comment]
@bioconda/core Hello, thank you for providing such a great resource. Please could you include my package Tapestry? Also, please send me an invitation link to join Bioconda. Many thanks.

File: recipes/tapestry/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+mkdir -p $PREFIX/bin
+cp $SRC_DIR/weave $PREFIX/bin
+cp $SRC_DIR/clean $PREFIX/bin
```
Comment (dpryan79) [inline recipes/tapestry/build.sh]: Is there a reason that installing the package doesn't install these?

File: recipes/tapestry/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - plumbum
+    - pysam
+    - scikit-learn >=0.2.0
+    - sqlalchemy
+    - tqdm
+
+test:
+  commands:
+    - weave -h &> /dev/null
+    - clean -h &> /dev/null
```
Comment (dpryan79) [inline recipes/tapestry/meta.yaml]: Do these import the module? If so that's fine, otherwise add an import test too.

**Comment by dpryan79** [comment]
Looks good, just see my comments.

**Comment by dpryan79** [comment]
@bioconda-bot recheck

File: recipes/tapestry/build.sh (hunk 1 above)
Comment (johnomics) [reply recipes/tapestry/build.sh]: Thanks for the rapid replies! Just my ignorance, I have now updated the setup.py to include the scripts and removed these lines from build.sh.

File: recipes/tapestry/meta.yaml (hunk 2 above)
Comment (johnomics) [reply recipes/tapestry/meta.yaml]: Yes, weave imports the module.

**Comment by dpryan79** [comment]
You'll need to redo the tag on github and update the sha256 for the tarball.

**Comment by johnomics** [comment]
Ah, thanks - redid the tag but not the sha256. Should be done now.

**Comment by dpryan79** [comment]
@bioconda-bot recheck

**Comment by dpryan79** [comment]
Great, can you tag a non-beta release so this is official? We try to avoid prereleases whenever possible.

**Comment by johnomics** [comment]
Okay, done - sorry about that.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Comment by dpryan79** [comment]
Thanks @johnomics!

**Comment by dpryan79** [comment]
BTW, shall I add you to bioconda so you can more easily maintain this?

**Comment by johnomics** [comment]
Thank you for your excellent work here! Yes, please add me to bioconda.

**Pull request merged**